### ISSUE TITLE
The status of refresh button in ‘Manage NuGet Packages’ window is incorrect after pressing

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#3266]
**Repro Steps:**
1. New project -> Other Languages ->Visual C# ->Windows ->Universal ->Blank App (Universal Windows).
2. Right-click on the project and click ‘Manage NuGet Packages…’.
3. Open ‘Browse’ tab and press refresh button.
4. Don't hover on the refresh button.

**Expected Result:**
The status of refresh button should be back to the status before pressing (not pressed down).

**Actual Result:**
The status of refresh button is incorrect (pressed down, blue background) as below screenshot. The status can be correct after you clicking other tabs (Installed/Updates).
![refreshbuttonstatusisincorrect](https://cloud.githubusercontent.com/assets/20632625/17322857/42deb926-58d2-11e6-853f-6352e96eeb5b.png)
**Note:**
It also repro on Dev14 U3+NuGet 1704.

**Comment by v-luzh** [comment]
**Note:**
1.It is found on Dev15 Preview4 25601.01.
2.It repro on Dev14 U3 (25420.01.00)+VSIX 3.5.0.1704.
3.It repro on NuGet 3.4.4.

**Comment by zkat** [comment]
@v-luzh does this still repro on 17.1?

**Comment by donnie-msft** [comment]
@zkat yes it does.

**Comment by v-luzh** [comment]
@zkat Yes, it still repro on  Main\31820.317 + NuGet Client Dev\6.1.0.30.

**Comment by v-luzh** [comment]
It still reproes on VS Main\35221.199 + NuGet Client Dev\6.12.0.81.
